### ISSUE TITLE
DP could not be resolved for current record in VCF

### ISSUE THREAD

**Issue by thomasyu888** [genome-nexus/annotation-tools#11]
I changed the values I bit here, but will send the vcf tomorrow.

```
Searching for files with extensions: vcf, maf, txt 
Loading data from file: error/GENIE-...
DP could not be resolved for current record in VCF: {'CHROM': '22', 'POS': '1323', 'ID': '.', 'REF': 'TG', 'ALT': 'T', 'QUAL': '.', 'FILTER': 'PASS', 'INFO': {'SOMATIC': '', 'AC': '1', 'AN': '4', 'END': '29682709', 'HOMLEN': '0', 'SVLEN': '-1', 'SVTYPE': 'DEL', 'ANNOVAR_DATE': '2018-04-16', 'Func.refGene': 'intronic', 'Gene.refGene': 'EWSR1', 'GeneDetail.refGene': '', 'ExonicFunc.refGene': '', 'AAChange.refGene': '', 'ExAC_ALL': '', 'ExAC_AFR': '', 'ExAC_AMR': '', 'ExAC_EAS': '', 'ExAC_FIN': '', 'ExAC_NFE': '', 'ExAC_OTH': '', 'ExAC_SAS': '', 'gnomAD_exome_ALL': '', 'gnomAD_exome_AFR': '', 'gnomAD_exome_AMR': '', 'gnomAD_exome_ASJ': '', 'gnomAD_exome_EAS': '', 'gnomAD_exome_FIN': '', 'gnomAD_exome_NFE': '', 'gnomAD_exome_OTH': '', 'gnomAD_exome_SAS': '', 'esp6500siv2_all': '', '1000g2015aug_all': '', 'ucsf500normT': '0.023', 'ucsf500normN': '0.00464', 'ALLELE_END': ''}, 'FORMAT': ['GT', 'AD', 'GMIMUT', 'GMIMAF', 'GMICOV', 'CallHC', 'CallUG', 'CallFB', 'CallPI', 'CallSID', 'CallMU', 'LR'], 'GENIE-...': '0/1:378,22:22:6:400:.:.:.:1:.:.:1.30598', 'GENIE-UCSF-1041-1213N': '0/0:.:.:.:.:.:.:.:.:.:.:0.0779254', 'MAPPED_TUMOR_FORMAT_DATA': {'GT': '0/1', 'AD': '378,25', 'GMIMUT': '32', 'GMIMAF': '8', 'GMICOV': '400', 'CallHC': '', 'CallUG': '', 'CallFB': '', 'CallPI': '1', 'CallSID': '', 'CallMU': '', 'LR': '1.30598', 'DP': '400.0'}, 'MAPPED_NORMAL_FORMAT_DATA': {'GT': '0/0', 'AD': ',', 'GMIMUT': '', 'GMIMAF': '', 'GMICOV': '', 'CallHC': '', 'CallUG': '', 'CallFB': '', 'CallPI': '', 'CallSID': '', 'CallMU': '', 'LR': '0.0779254'}}
```

**Comment by sheridancbio** [comment]
This error has been reproduced, and is being investigated. The parsing of the contained tumor sample appears to be successfully resolved for depth (DP=240), but the normal sample does not receive any computed DP value. Parsed values for comparison:

**Comment by sheridancbio** [comment]
'GENIE-SAGE-1-T': '0/1:209,31:31:13:240:.:.:.:1:.:.:1.04146'
'GENIE-SAGE-1-N': '0/0:.:.:.:.:.:.:.:.:.:.:0.0779254'

**Comment by sheridancbio** [comment]
On first inspection, it seems that indeed the normal sample depth is not present in the source file.(AD=.)
Comparison to the behavior under vcf2maf should be done ... if vcf2maf does not report an error when parsing the file and is able to output a record where the normal sequencing count field is left blank then it may be that error detection logic here is too strict.

**Comment by thomasyu888** [comment]
I don't think vcf2maf throws an error for these vcfs.

**Comment by ao508** [comment]
if depth is undefined for the normal as it is in this case then  the VCF empty value is used (".") [see here](https://github.com/mskcc/vcf2maf/blob/master/vcf2maf.pl#L817) - in this case we should use our MAF default value when a datum is missing (empty string) - making the fix in this pr #13

**Comment by ao508** [comment]
@thomasyu888 This issue has been resolved in the latest version of the master branch. Let us know if it persists.
